pwiz_tools/Skyline/Controls/Graphs/DetectionsPlotPane.cs

```diff
@@ -46,12 +46,12 @@ public abstract class DetectionsPlotPane : SummaryReplicateGraphPane
 
         protected DetectionsPlotPane(GraphSummary graphSummary) : base(graphSummary)
         {
-            MaxRepCount = graphSummary.DocumentUIContainer.DocumentUI.MeasuredResults.Chromatograms.Count;
+            MaxRepCount = graphSummary.DocumentUIContainer.DocumentUI?.MeasuredResults?.Chromatograms?.Count ?? 0;

```

Comment:
I think the only "?" that is necessary is the one between "DocumentUI" and "MeasuredResults".
The other "?" characters should probably be removed.

Reply:
Sure. I wasn't sure if ?. is short-circuiting.